crates/router/src/connector/trustpay.rs

```diff
@@ -669,12 +669,17 @@ impl api::IncomingWebhook for Trustpay {
             (trustpay::CreditDebitIndicator::Dbit, trustpay::WebhookStatus::Chargebacked) => {
                 Ok(api_models::webhooks::IncomingWebhookEvent::DisputeLost)
             }
-       
-            (trustpay::CreditDebitIndicator::Dbit | trustpay::CreditDebitIndicator::Crdt, trustpay::WebhookStatus::Unknown  )   => Ok(api::IncomingWebhookEvent::EventNotSupported),
-            (trustpay::CreditDebitIndicator::Crdt, trustpay::WebhookStatus::Refunded) => Ok(api::IncomingWebhookEvent::EventNotSupported),
-            (trustpay::CreditDebitIndicator::Crdt, trustpay::WebhookStatus::Chargebacked) =>Ok(api::IncomingWebhookEvent::EventNotSupported),
-
 
+            (
+                trustpay::CreditDebitIndicator::Dbit | trustpay::CreditDebitIndicator::Crdt,
+                trustpay::WebhookStatus::Unknown,
+            ) => Ok(api::IncomingWebhookEvent::EventNotSupported),
+            (trustpay::CreditDebitIndicator::Crdt, trustpay::WebhookStatus::Refunded) => {
+                Ok(api::IncomingWebhookEvent::EventNotSupported)
+            }
+            (trustpay::CreditDebitIndicator::Crdt, trustpay::WebhookStatus::Chargebacked) => {
+                Ok(api::IncomingWebhookEvent::EventNotSupported)
+            }
```

Comment:
```suggestion
            (trustpay::CreditDebitIndicator::Crdt, trustpay::WebhookStatus::Refunded | trustpay::WebhookStatus::Chargebacked) => {
                Ok(api::IncomingWebhookEvent::EventNotSupported)
            }
```

You may have to run the formatter.